client/components/account-status/style.scss

```diff
@@ -59,11 +65,11 @@
 		}
 	}
 
-	@media only screen and ( max-width: 400px ) {
+	@media only screen and ( max-width: 390px ) {
 		div:first-child {
```

Comment:
We should avoid styling tags directly, as it can cause conflicts, giving a class then styling is the preferred way, can you please check if this can be changed.

Reply:
Nit: we can store 390px as a variable and use it in multiple places.

`$mobile-break-point-390: 390px;`